main.go

```diff
@@ -80,8 +82,13 @@ func main() {
 		os.Exit(1)
 	}
 
-	var controller *controllers.WavefrontReconciler
-	controller, err = controllers.NewWavefrontReconciler(version, mgr.GetClient(), mgr.GetScheme())
+	objClient, err := client.New(config, client.Options{Scheme: scheme})
```

Comment:
We need our own client for the reconciler since the Manager client is scoped only to the operator's namespace.